File: Backend/MirTech/MirTech/Entity/EntityModels/Employee.cs
namespace MirTech.Entity.EntityModels;

public class Employee
{
    public int Id { get; set; }
    public string Department { get; set; }
    public string FullName { get; set; }
    public DateTime Birthday { get; set; }
    public DateTime HireDate { get; set; }
    public int Salary { get; set; }
}

File: Frontend/MirTech/src/classes/Employee.ts
export class Employee{
    id?: number;
    fullName?: string = "";
    department?: string = "";
    birthday?: Date = new Date();
    hireDate?: Date = new Date();
    salary?: number = 0;

    constructor(id?: number, fullName?: string, department?: string, birthday?: Date, hireDate?: Date, salary?: number){
        this.id = id;
        this.fullName = fullName;
        this.department = department;
        this.birthday = birthday;
        this.hireDate = hireDate;
        this.salary = salary;
    }
}